package.json

```diff
@@ -65,6 +65,7 @@
     "serial": true
   },
   "dependencies": {
+    "@streamlabs/obs-studio-node": "https://github.com/stream-labs/obs-studio-node/releases/download/v0.3.28/iojs-v3.0.4-signed.tar.gz",
```

Comment:
@mbiemiller is there a reason you're making this change to the `package.json`? it'll definitely have an effect on the yarn lock